src/hooks/useQuizState/useQuizApiState.ts

```diff
@@ -60,6 +65,16 @@ const useQuizApiState: UseQuizApiState = (
     }
   };
 
+  const dispatchQuizResultsConfig = async () => {
+    const quizResultsConfig = await getQuizResultsConfig(cioClient, quizId, {
```

Comment:
Can we wrap this in a try catch please

Reply:
[67cbc4c](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/155/commits/67cbc4ca06d9703347e8657339d6cbc7db350f44)